### ISSUE TITLE
[typescript] Add scrpt to watch and serve in parallel

### ISSUE THREAD

**Issue by tbuschto** [eclipsesource/generator-tabris-js#31]
It would be really not having two run two tools independently to development typescript: watching for changes and the http-server.

This can be achieved like this:

```
{
  "name": "hello-world",
  "version": "0.1.0",
  "main": "dist/app.js",
  "scripts": {
    "build": "tsc -p .",
    "watch": "tsc -w -p .",
    "start": "npm-run-all --parallel watch serve",
    "serve": "tabris serve"
  },
  "dependencies": {
    "tabris": "^2.0.0"
  },
  "devDependencies": {
    "typescript": "^2.2.1",
    "npm-run-all": "^4.0.2"
  }
}
```

Note `    "npm-run-all": "^4.0.2"`!

**Comment by tbuschto** [comment]
Was already done at one point
